### DISCUSSION TITLE
foreign table RLS help

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#6193] [Questions] [answered]
I want to set SELECT RLS on messages table that only users who are involved in current conversation can select messages.

![image](https://user-images.githubusercontent.com/56837461/161006763-d3f9d853-d687-4e6f-a07e-347e647aedc1.png)

how i can get conversation_id from messages table?

**Comment by GaryAustin1** [comment, marked as answer]
Your conversations.id = messages.conversation_id line looks correct if that policy is on messages table and you have conversation_id column in the messages table.

**Reply by EdgarsLv** [reply]
hmm strange, was trying to save like that and got error something about messages unknown... now tried saving and it worked.
